src/main/scala/io/github/spark_redshift_community/spark/redshift/Conversions.scala

```diff
@@ -73,6 +82,20 @@ private[redshift] object Conversions {
     new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS")
   }
 
+  def parseRedshiftTimestamp(s: String): Timestamp = {
+    val temporalAccessor = formatter.parse(s)
+
+    try {
+      // timestamptz
+      Timestamp.from(ZonedDateTime.from(temporalAccessor).toInstant)
+    }
+    catch {
+      // Case timestamp without timezone
+      case e: DateTimeException =>
+        Timestamp.valueOf(LocalDateTime.from(temporalAccessor))
```

Comment:
I am uncertain of this behavior because LocalDateTime will influence by the Spark local timezone setting. When Redshift only has timestamp (within timezone data). 

https://docs.aws.amazon.com/redshift/latest/dg/r_Datetime_types.html

Redshift docs says, "By default, TIMESTAMP values are Coordinated Universal Time (UTC) in both user tables and Amazon Redshift system tables."

Reply:
never mind, i thought LocalDateTime just use the local timezone data from the machine. The class is defined as "This class does not store or represent a time-zone. "